Stripe/PublicHeaders/STPPaymentConfiguration.h

```diff
@@ -116,6 +116,15 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, nullable, copy) NSURL *returnURL  NS_EXTENSION_UNAVAILABLE("Redirect based sources are not available in extensions");
 
+
+/**
+ *  For payment methods that redirect a user to authorize a payment, the SDK
+ *  will poll the source for up to this amount of time after the user returns to
+ *  the app in order to determine the status of the payment. The default is 10
+ *  seconds.
+ */
+@property (nonatomic) NSTimeInterval pollingTimeout;
```

Comment:
There's a built in maximum to the poller timeout right? Should mention that here as well, or refer to that other api doc.